src/site/stages/build/drupal/graphql/nodeStepByStep.graphql.js

```diff
@@ -52,6 +52,27 @@ fragment nodeStepByStep on NodeStepByStep {
       }
     }
   }
+  fieldRelatedBenefitHubs {
+    entity {
+      ... on NodeLandingPage {
+        fieldSupportServices {
+          entity {
+            ... on NodeSupportService {
+              title
+              fieldLink {
+                title
+                url {
+                  path
+                  routed
+                }
+              }
+              fieldPhoneNumber
+            }
+          }
+        }
+      }
+    }
+  }
```

Comment:
We could clean up some of this duplication by creating a new fragment `... on NodeSupportService`. What do you think?

Reply:
Totally, one sec!